### ISSUE TITLE
Page number and total number

### ISSUE THREAD

**Issue by caramelopardalis** [caramelopardalis/browser-report#9]

**Comment by caramelopardalis** [comment]
Web page can't render content on print margin.

![print-margin-is-untouchable](https://user-images.githubusercontent.com/1132197/50467511-b082c900-09e6-11e9-8c1f-ecce78885361.jpg)

**Comment by caramelopardalis** [comment]
So added pseudo print margin.

![pseudo-print-margin](https://user-images.githubusercontent.com/1132197/50468361-c2666b00-09ea-11e9-9d2b-a82f5b1bf385.jpg)

**Comment by caramelopardalis** [comment]
Now this is implemented (846d081f1a94eef1fdb944e3176c8c22225c5b60).

You can use `.br-page-margin-top`, `.br-page-margin-bottom`, `.br-page-number` and `.br-total-number` clasesses like below. `.br-page-margin-top` and `.br-page-margin-bottom` are pseudo print margin.

```html
<div class="br-content">
    <div class="br-page-margin-top">
        ：8]：8]：8]：8]：8]：8]
    </div>
    <div class="br-page-header">
        <h1>Gorilla Report</h1>
    </div>
    <div class="br-page-footer">
        <q>https://en.wikipedia.org/wiki/Western_lowland_gorilla</q><br>
    </div>
    <div class="br-page-margin-bottom">
        <div class="page-numbers">
            <span class="br-page-number"></span> / <span class="br-total-number"></span>
        </div>
    </div>
</div>
```
